crates/bevy_winit/src/system.rs

```diff
@@ -81,6 +84,22 @@ pub(crate) fn create_windows<'a>(
                 window: window.clone(),
             });
 
+        #[cfg(target_arch = "wasm32")]
+        {
+            if window.fit_canvas_to_parent {
+                let canvas: HtmlCanvasElement = web_sys::window()
+                    .unwrap()
+                    .document()
+                    .unwrap()
+                    .query_selector("canvas")
```

Comment:
This is probably wrong, we should check for Window::canvas ; or retrieve it in some other way.

Also, that's quite a lot of unwraps 🤔